snuba/subscriptions/data.py

```diff
@@ -111,10 +120,77 @@ def to_dict(self) -> Mapping[str, Any]:
         raise NotImplementedError
 
 
+@dataclass(frozen=True, kw_only=True)
+class RPCSubscriptionData(SubscriptionData):
+    """
+    Represents the state of an RPC subscription.
+    """
+
+    table_request: str
+
```

Comment:
1. the table_request is bytes, not a string
2. we should be storing the name of the endpoint and the version of it. The API will evolve and we need to be able to change the model along with it